nfs-repo-and-addon.ks.in

```diff
@@ -1,106 +0,0 @@
-#test name: nfs-repo-and-addon
```

Comment:
Please explain (in the commit message?) why this test is deleted. I can tell that it does not fit into the set, but still...